test/Interop/C/function/emit-called-inline-function-irgen.swift

```diff
@@ -13,10 +13,9 @@ import EmitCalledInlineFunction
 // C99-DAG: define internal i32 @calledFromSwift()
 // C99-DAG: define internal i32 @calledTransitively()
 
-// CXX-DAG: define linkonce_odr{{( dso_local)?}} i32 @{{_Z15calledFromSwiftv|"\?calledFromSwift@@YAHXZ"}}()
-// CXX-DAG: define linkonce_odr{{( dso_local)?}} i32 @{{_Z18calledTransitivelyv|"\?calledTransitively@@YAHXZ"}}()
-// CXX-DAG: define linkonce_odr{{( dso_local)?}} i32 @{{_ZN1C32memberFunctionCalledTransitivelyEv|"\?memberFunctionCalledTransitively@C@@QEAAHXZ"}}(%class.C* nonnull align 1 dereferenceable(1) %this)
-// CXX-DAG: define linkonce_odr{{( dso_local)?}} i32 @{{_Z29calledTransitivelyFromVarInitv|"\?calledTransitivelyFromVarInit@@YAHXZ"}}()
```

Comment:
Where'd this one go?

Reply:
... 🤔 that's a very good question. Did I accidentally `dd`? How strange.